libnavigation-core/src/main/java/com/mapbox/navigation/core/directions/session/DirectionsSession.kt

```diff
@@ -12,16 +11,14 @@ internal interface DirectionsSession {
 
     fun requestRoutes(routeOptions: RouteOptions, routesRequestCallback: RoutesRequestCallback)
 
+    fun requestFasterRoute(adjustedRouteOptions: RouteOptions, routesRequestCallback: RoutesRequestCallback)
```

Comment:
Can we add a doc here that this method does the same thing that `requestRoutes`, but ignores the result and it's up to the consumer to take the action with the route?